source/faucet/config.yaml

```diff
@@ -4,6 +4,8 @@
 
 # When we deploy Faucet on the real TestNet, the keys used will not be well-known.
 keys:
+  # Genesis seed
+  - SDN7BBGE6Z6OQM3K4PACLTZUJ5QX4AY4QPDQ2JJ2JCFWCG2OIYYALIRY
```

Comment:
Perhaps the comment can go at end of line to make it clear it is just that line.

Reply:
Although maybe it is not possible.

Reply:
Or another comment for the 6 keys below?